packages/realm/RealmJS.podspec

```diff
@@ -35,36 +34,48 @@ Pod::Spec.new do |s|
   s.homepage               = package['homepage']
   s.platform               = :ios, '9.0'
 
-  # The source field is a required field in the podspec, but it is not ment to be used.
-  # This is because the Podspec is not ment to be published into a CocoaPod repository, instead React Native uses a :path style dependency when adding this to the users projects Podfile.
+  # The source field is a required field in the podspec, but it is not meant to be used.
+  # This is because the Podspec is not meant to be published into a CocoaPod repository, instead React Native uses a :path style dependency when adding this to the users projects Podfile.
   # @see https://guides.cocoapods.org/using/the-podfile.html#using-the-files-from-a-folder-local-to-the-machine
   # @see https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#platform-ios
   s.source                 = { :http => 'https://github.com/realm/realm-js/blob/main/CONTRIBUTING.md#how-to-debug-react-native-podspec' }
 
-  s.source_files           = 'react-native/ios/RealmReact/*.mm'
-  s.public_header_files    = 'react-native/ios/RealmReact/*.h'
+  s.source_files           = 'binding/jsi/*.cpp',
+                             'binding/apple/*.mm'
+
+  s.public_header_files    = 'binding/apple/*.h'
 
   s.frameworks             = uses_frameworks ? ['React'] : []
 
   s.library                = 'c++', 'z', 'compression'
 
   s.pod_target_xcconfig    = {
                                 # Setting up clang
-                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
+                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++20',
```

Comment:
We're using C++20 features in the binding.

Reply:
Should we merge or close https://github.com/realm/realm-js/pull/5053?

Reply:
Perhaps 🤔 I haven't double checked the current state of the Node.js binding.